docs/bundle.js

```diff
@@ -295,6 +295,9 @@
 	(0, _reactDom.render)(_react2.default.createElement(
```

Comment:
Assuming . you ran a `make docs` command to generate this file?

Reply:
Yep! Is running `make docs` necessary? It wasn't quite clear to me

Reply:
I'm pretty sure! Although I don't know the docs very well in this repo tbh...